addon/components/lazy-popover-on-element.js

```diff
@@ -0,0 +1,129 @@
+import Ember from 'ember';
```

Comment:
no changes to this file except the addition of the `didRender`. Appears changed due to `git mv`.